circuit_knitting/cutting/__init__.py

```diff
@@ -38,6 +38,23 @@
     instructions.CutWire
     instructions.Move
 
+Automatic Cut Finding
+~~~~~~~~~~~~~~~~~~~~~
+
+.. autosummary::
+    :toctree: ../stubs/
+    :nosignatures:
+
+    find_cuts
+
+.. autosummary::
+    :toctree: ../stubs/
+    :nosignatures:
+    :template: autosummary/class_no_inherited_members.rst
+
+    automated_cut_finding.OptimizationParameters
+    automated_cut_finding.DeviceConstraints
```

Comment:
I tried this locally; it works.

```suggestion
    OptimizationParameters
    DeviceConstraints
```

Reply:
Would you want to go ahead and make this change or would you prefer I did it?

Reply:
The suggestion here would make the sphinx output look a bit nicer.

#### Before
![sphinx2](https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/assets/91987/888cd170-c80c-4b9a-8aad-5e9ae84d85f2)
#### After
![sphinx1](https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/assets/91987/a9204f13-0fe0-446c-9472-c6684da7cf5e)